target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-haze.dts

```diff
@@ -303,3 +310,41 @@
 
 	qcom,ath11k-calibration-variant = "prpl-Haze";
 };
+
+&blsp1_i2c3{
+	pinctrl-0 = <&i2c_3_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	led-controller@30 {
+		compatible = "ti,lp5562";
+		reg = <0x30>;
+		clock-mode = /bits/ 8 <2>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		status_red: chan@0 {
+			chan-name = "red";
+			led-cur = /bits/ 8 <0x20>;
+			max-cur = /bits/ 8 <0x60>;
+			color = <LED_COLOR_ID_RED>;
+			reg = <0>;
+		};
+
+		status_green: chan@1 {
+			chan-name = "green";
+			led-cur = /bits/ 8 <0x20>;
+			max-cur = /bits/ 8 <0x60>;
+			color = <LED_COLOR_ID_GREEN>;
+			reg = <1>;
+		};
+
+		status_blue: chan@2 {
```

Comment:
Nitpick, there seems to be a convention to name the aliases in the following way

```suggestion
		led_system_blue: chan@2 {
```